### ISSUE TITLE
missing ENVIRONMENT_VARIABLE_TEMP

### ISSUE THREAD

**Issue by tfarina** [damian-m-g/xlsx_drone#3]
Program received signal SIGSEGV, Segmentation fault.
__strlen_sse2_bsf () at ../sysdeps/i386/i686/multiarch/strlen-sse2-bsf.S:50
50	../sysdeps/i386/i686/multiarch/strlen-sse2-bsf.S: No such file or directory.
(gdb) p temp_path
No symbol "temp_path" in current context.
(gdb) p temp_folder
No symbol "temp_folder" in current context.
(gdb) up
#1  0x0804b3ec in xlsx_open (src=0x8069b08 "test.xlsx", xlsx=0xbfffeea4) at ../third_party/xlsx_drone/xlsx_drone.c:54
54	  int deployed_xlsx_path_len = strlen(temp_path) + strlen(temp_folder);
(gdb) p temp_folder
$1 = 0xb7fb580c <tmpnam_buffer> "/tmp/fileDVib7r"
(gdb) p temp_path
$2 = 0x0

I don't have this variable in my environment.

**Comment by damian-m-g** [comment]
Hi @tfarina, I've implemented a simple fix. As far as I understand, it's rare to not have those env variables defined, but in that case POSIX/FHS says that "/tmp" is the root for temporary files so I'm hardcoding that in case those env vars aren't defined. Commit https://github.com/damian-m-g/xlsx_drone/commit/16693c84b52a882b89513c61a7e04da7ecf1f907 does it, and also the last release (0.2.3). Thanks for the tips on how to handle the temp dir issue, I've made a copy of that snippet for future needs. Cheers.
